### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by qwjaskzxl**
fix 2 bugs in "models and schedulers toolbox" Quickstart example codes: (1) "previous_noisy_sample" should be in the FOR loop in line 88; (2) converting image to INT should be before "Image.fromarray" in line 93

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,13 +84,11 @@ input = noise
 for t in scheduler.timesteps:
     with torch.no_grad():
         noisy_residual = model(input, t).sample
-
-previous_noisy_sample = scheduler.step(noisy_residual, t, input).prev_sample
-input = previous_noisy_sample
+        previous_noisy_sample = scheduler.step(noisy_residual, t, input).prev_sample
```
Comment (patrickvonplaten) [inline README.md:87]:
```suggestion
        prev_noisy_sample = scheduler.step(noisy_residual, t, input).prev_sample
        input = prev_noisy_sample
```

**Comment by patrickvonplaten** [comment]
Thanks a lot for the PR @qwjaskzxl - @yiyixuxu could you maybe quickly run all the code of the main README.md to make sure it works as expected?

**Comment by patrickvonplaten** [comment]
Also cc @stevhliu

**Comment by stevhliu** [comment]
Ah my bad! Thanks for catching the bugs, all the code should work now.

**Review by yiyixuxu — APPROVED** [review]
Thanks!

**Comment by qwjaskzxl** [comment]
@patrickvonplaten Yeah, u r right. All the code of the main README.md works well.

**Comment by patrickvonplaten** [comment]
Thansk a lot for the bug fix @qwjaskzxl !

**Pull request merged**